<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrogramDialog</class>
 <widget class="QDialog" name="SpectrogramDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="Layout_mainWidget" stretch="1,3">
   <item>
    <widget class="QWidget" name="widget_functionArea" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_changeAxis">
        <property name="title">
         <string>Change Axis</string>
        </property>
        <layout class="QGridLayout" name="Layout_ChangeAxis">
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="Layout_XYAxisInPut">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>X:start</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_xStart"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>end</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEdit_yEnd"/>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_xEnd"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_yStart"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>end</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Y:start</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_Change">
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>93</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_DisplayEnhancement">
        <property name="title">
         <string>Display enhancement</string>
        </property>
        <layout class="QHBoxLayout" name="Layout_DisplayEnhancement">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Enhance</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>75</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Enhance">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_DisplayOptimization">
        <property name="title">
         <string>Display Optimization </string>
        </property>
        <layout class="QGridLayout" name="Layout_DisplayOptimization">
         <item row="0" column="0" colspan="3">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Background Color Select:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBox_BackgroundColorSelect"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Noise Data Clear Mode:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBox_DataClearMode"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Clear Start In:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_Remove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>9</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pushButton_Back">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Layout_Openbutton">
        <item>
         <widget class="QPushButton" name="pushButton_OpenFile">
          <property name="text">
           <string>Open file</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_SaveSpectrogram">
          <property name="text">
           <string>Save Spectrogram</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_drawArea" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
